sympy/solvers/solvers.py

```diff
@@ -470,26 +502,28 @@ def solve(f, *symbols, **flags):
         >>> solve(x + exp(x)**2, exp(x), set=True)
         ([exp(x)], {(-sqrt(-x),), (sqrt(-x),)})
 
-        >>> from sympy import Indexed, IndexedBase, Tuple, sqrt
+        >>> from sympy import Indexed, IndexedBase, Tuple
         >>> A = IndexedBase('A')
         >>> eqs = Tuple(A[1] + A[2] - 3, A[1] - A[2] + 1)
         >>> solve(eqs, eqs.atoms(Indexed))
         {A[1]: 1, A[2]: 2}
 
-        * To solve for a symbol implicitly, use implicit=True:
+        * To solve for a function within a derivative, use ``dsolve``.
```

Comment:
```suggestion
        * To solve for a function within a derivative, use :func:`~.dsolve`.
```

to provide a link to the function mentioned